docs/appkit/features/bitcoin.mdx

```diff
@@ -0,0 +1,59 @@
+---
+pagination_next: appkit/react/core/installation
+title: Bitcoin
+---
+
+import Button from '../../components/button'
+import Container from '../../components/Container.js'
+import Wrapper from '../../components/Home/Wrapper'
+import W3MQuickStart from '../../components/W3MQuickStart'
+import useBaseUrl from '@docusaurus/useBaseUrl'
+
+import reactLogo from '../../../static/assets/home/reactLogo.png'
+import nextjsLogo from '../../../static/assets/home/nextjsLogo.png'
+import vueLogo from '../../../static/assets/home/vueLogo.png'
+import javascriptLogo from '../../../static/assets/home/javascriptLogo.png'
+
+# Bitcoin
+
+The AppKit SDK supports Bitcoin, allowing users to connect their Bitcoin wallets to applications. AppKit provides a simple, secure, and seamless in-app experience for users looking to transact within web3.
```

Comment:
We should add more content here. 

It would be great if we can add a brief paragraph on how fragmented things are when it comes to building on top of Bitcoin. Then we can mention how AppKit solves this problem in the Bitcoin ecosystem. 

I can work on this

Reply:
yes, it was a really first version. we should be adding more context here and also in solana